AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1645,17 +1647,12 @@ - (void)recursivelyClearFetchedData
 }
 
 /**
- * We currently only set interface state on nodes
- * in table/collection views. For other nodes, if they are
- * in the hierarchy we return `Unknown`, otherwise we return `None`.
- *
- * TODO: Avoid traversing up node hierarchy due to possible deadlock.
- * @see https://github.com/facebook/AsyncDisplayKit/issues/900
- * Possible solution is to push `isInCellNode` state downward on `addSubnode`/`removeFromSupernode`.
+ * We currently only set interface state on nodes in table/collection views. For other nodes, if they are
+ * in the hierarchy we enable all ASInterfaceState types with `ASInterfaceStateInHierarchy`, otherwise `None`.
  */
-- (BOOL)supportsInterfaceState {
-  return ([self isKindOfClass:ASCellNode.class]
-      || [self _supernodeWithClass:ASCellNode.class checkViewHierarchy:NO] != nil);
+- (BOOL)supportsRangeManagedInterfaceState
```

Comment:
What's the future plan for this method? Will it be removed and all nodes support interface state (considering that range managed will be one part of the interface state feature set)?